src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -366,6 +366,30 @@ void LottieParser::parseSlotProperty(T& prop)
     }
 }
 
+void LottieParser::parseSlotImage(LottieImage* obj)
+{
+    unsigned long id = 0;
+    const char* data = nullptr;
+    const char* subPath = nullptr;
+    float width = 0.0f;
+    float height = 0.0f;
+    auto embedded = false;
+
+    // currently only supports image
+    while (auto key = nextObjectKey()) {
```

Comment:
Please avoid adding duplicate parsing logic

Reply:
Remove `parseSlotImage` and use `parseAsset` function